transcription/keys.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
For now this is okay as this model's accuracy is low and we will also need to consider other database templates, but we will need a more comprehensive list of keys for all subheadings in log template